docs/notebooks/survival_model.ipynb

```diff
@@ -0,0 +1,1196 @@
+{
```

Comment:
<p>The text is excellent so far. The only thing I want to point is that you have three different expressions where you write the same on the left, but something different on the right. I think that should be changed.</p><p> </p><p>**Edit** a minor comment is that I don't like talking about "Censored" as a distribution because it's not. It's an utility that helps you to create censored distributions. But since PyMC calls it a distribution, let's leave it as it is.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Yeah that's a good point. I updated so PDF uses $f(D_i | \lambda_i)$, CDF uses $F(D_i | \lambda_I)$, and the CCDF uses $S(D_i|\lambda) = \text{exp}(-\lambda_i D_I)$. CCDF uses $S$ because the survival function can be written as the CCDF.</p>